lib/rebaser-server/src/change_set_requests/handlers.rs

```diff
@@ -94,6 +94,7 @@ pub async fn process_request(State(state): State<AppState>, msg: InnerMessage) -
     if RebaseStatusDiscriminants::Success == rebase_status.clone().into() {
         if let Some(workspace) = Workspace::get_by_pk(&ctx, &workspace_pk.into()).await? {
             if workspace.default_change_set_id() == ctx.visibility().change_set_id {
+                ctx.update_snapshot_to_visibility().await?;
```

Comment:
The actions were only being dispatched the second time they appeared in a commit to head, since the first time the rebaser had added them to head, but the ctx didn't have them since it wasn't in the latest snapshot